gammapy/irf/psf_analytical.py

```diff
@@ -354,3 +358,46 @@ def info(self, fractions=[0.68, 0.95], energies=Quantity([1., 10.], 'TeV'),
                            "E = {2:4.1f}: {3:5.8f}\n"
                            "".format(100 * fraction, theta, energy, radius))
         return ss
+
+
+
+    def to_table_psf(self, theta=None, offset_max=None, binsz=0.01, exposure=None):
+        """
+        Convert triple Gaussian PSF ot table PSF.
+
+        Parameters
+        ----------
+        theta : `~astropy.coordinates.Angle`
+            Observation offset. Default theta = 0 deg.
+        offset_max : `~astropy.coordinates.Angle`
+            Maximum offset from PSF center. Default offset_max = 2.5 deg
+        binsz : float
+            Size of the spatial bins in deg / pixel. Default binsz = 0.01 deg
+        exposure : `~astropy.units.Quantity`
+            Energy dependent exposure. Should be in units equivalent to 'cm^2 s'.
+
+        Returns
+        -------
+        tabe_psf : `~gammapy.irf.EnergyDependentTablePSF`
+            Instance of `EnergyDependentTablePSF`.
+        """
+        # Convert energies to log center
+        ebounds = EnergyBounds.from_lower_and_upper_bounds(self.energy_lo, self.energy_hi)
+        energies = ebounds.log_centers
+        
+        # Default for exposure
+        exposure = exposure or Quantity(np.ones(len(energies)), 'cm^2 s')
+        theta = theta or Angle(0, 'deg')
+        offset_max = offset_max or Angle(2.5, 'deg')
+
+        offsets = Angle(np.arange(0, offset_max.degree, binsz), 'deg')
+
+        psf_value = Quantity(np.empty((len(energies), len(offsets))), 'deg^-2')
+
+        for i, energy in enumerate(energies):
+            psf_gauss = self.psf_at_energy_and_theta(energy, theta)
+            psf_value[i] = Quantity(psf_gauss(offsets, np.zeros_like(offsets)), 'deg^-2')
```

Comment:
@cdeil @adonath @registerrier 
For the moment in the method self.psf_at_energy_and_theta() there is no interpolation. You select the energy bin and offset bin that is the closest from the energy and theta that you gave. You select the parameters of the PSF like sigma and norm for the tripple gauss and then you create an EnergyDependentTablePSF with psf_gauss(offsets, np.zeros_like(offsets)). But in theta, you could have quite large variation in the PSF parametrisation so an interpolation would be better no?
For the interpolation, it could be very tricky to do it directly on the parameters of the triple gauss for example and I think even completely false. Maybe we could define 4 EnergyDependentTablePSF for the two energies and two theta closest to the energy and theta you give to self.psf_at_energy_and_theta. And then create an interpolator in (energy,theta) on these tables for each offset of the table? With this method, the interpolation will not depend on the profil you assumed for the PSF (tripple gauss, king ...etc)
I'm not sure to be really clear, this is difficult by writing...